requirements/base.in

```diff
@@ -7,3 +7,4 @@ Pillow                    # Image manipulation module, required to use ImageFiel
 django_extensions         # Required to use TimeStampedModel
 django-simple-history     # History for Django models
 django-object-actions     # Object actions in Django admin
+edx-rest-api-client       # For accessing the Enrollment API (and possibly other edX APIs)
```

Comment:
@mattdrayer The catalog API is currently accessed via a wrapper of ERAC in edx-platform, so the only immediate reason to have this requirement here is the Enrollment API client.